saga/sagalogs/inMemorySagaLog.go

```diff
@@ -1,8 +1,9 @@
-package saga
+package sagalogs
```

Comment:
You'll probably want to rename this to inmemory_saga_log.go (and rectify with Caitie's concurrent change....)